src/components/error-boundary.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+
+class ErrorBoundary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { hasError: false, error: null };
+  }
+
+  static getDerivedStateFromError(error) {
+    return { hasError: true, error };
+  }
+
+  componentDidCatch() {
+    // You can also log the error to an error reporting service
+    // See https://reactjs.org/docs/error-boundaries.html
```

Comment:
We need to do exactly what is proposed here: use Raven library (already in project) to report errors

Reply:
Done